app/models/rook.rb

```diff
@@ -1,2 +1,5 @@
 class Rook < Piece
+  # Can move horizontal or vertical for unlimited spaces unless path is blocked
+  def valid_move?(x, y)
+
```

Comment:
I think this will need another `end` on line 4 while the method is being built. Otherwise, attempts to display the rooks may raise errors.